sdk/communication/azure-communication-administration/samples/phone_number_area_codes_sample.py

```diff
@@ -0,0 +1,43 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: phone_number_area_codes_sample.py
+DESCRIPTION:
+    These samples demonstrate area codes samples.
+
+    ///getting all area codes via a connection string, country code and phone plan id
+USAGE:
+    python phone_number_area_codes_sample.py
```

Comment:
Let's list required environment variables here. 

You can take https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/metricsadvisor/azure-ai-metricsadvisor/samples/sample_data_feeds.py as a reference.